build/ci/.azure-devops-wasm-uitests.yml

```diff
@@ -147,22 +154,43 @@ jobs:
         UITEST_AUTOMATED_GROUP: 'Benchmarks'
         UITEST_TEST_TIMEOUT: '120000'
 
-      net7_runtimetests_group_00:
-        SITE_SUFFIX: 'net7.0'
+      net7_uwp_runtimetests_group_00:
+        SITE_SUFFIX: 'net7.0-UWP'
         UITEST_AUTOMATED_GROUP: 'RuntimeTests'
         UITEST_TEST_TIMEOUT: '1800000'
         UITEST_RUNTIME_TEST_GROUP: 0
         UITEST_RUNTIME_TEST_GROUP_COUNT: 3
 
-      net7_runtimetests_group_01:
-        SITE_SUFFIX: 'net7.0'
+      net7_uwp_runtimetests_group_01:
+        SITE_SUFFIX: 'net7.0-UWP'
         UITEST_AUTOMATED_GROUP: 'RuntimeTests'
         UITEST_TEST_TIMEOUT: '1800000'
         UITEST_RUNTIME_TEST_GROUP: 1
         UITEST_RUNTIME_TEST_GROUP_COUNT: 3
 
-      net7_runtimetests_group_02:
-        SITE_SUFFIX: 'net7.0'
+      net7_uwp_runtimetests_group_02:
+        SITE_SUFFIX: 'net7.0-UWP'
```

Comment:
How does this determine which version of the source to run? I see `SITE_SUFFIX` used to differentiate between the artifacts, but not sure how the "uwp" group will pick UWP build and "winui" group will pick the converted one 🤔

Reply:
It really seems both UWP and WinUI Runtime tests are downloading the same build artifacts unless I am looking in the wrong place 😀

Reply:
It could be :) I'll be fixing this once 4.6 is out :)

Reply:
Thinking about it, there's no need for conversion here, as the tooling only uses already built samples.